shared/modules/selectors/networks.test.ts

```diff
@@ -0,0 +1,253 @@
+import { AccountsControllerState } from '@metamask/accounts-controller';
+import { NON_EVM_TESTNET_IDS } from '@metamask/multichain-network-controller';
+import mockState from '../../../test/data/mock-state.json';
+import { CAIP_FORMATTED_EVM_TEST_CHAINS } from '../../constants/network';
+import {
+  getNonTestNetworks,
+  getNetworksByScopes,
+  MultichainNetworkConfigurationsByChainIdState,
+} from './networks';
+
+const typedMockState =
+  mockState as unknown as MultichainNetworkConfigurationsByChainIdState & {
+    metamask: {
+      internalAccounts: AccountsControllerState['internalAccounts'];
+    };
+  };
+
+const extendedMockState = {
```

Comment:
Wondering if it might be cleaner to just add this to `mock-state.json` so we can potentially re-use this dummy state elsewhere